insights-notification-schemas-java/src/main/java/com/redhat/cloud/notifications/ingress/Registry.java

```diff
@@ -0,0 +1,38 @@
+package com.redhat.cloud.notifications.ingress;
+
+import org.apache.avro.Schema;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Registry {
+
+    private final Map<String, Schema> avroSchemas = new HashMap<>();
```

Comment:
It's possible, should I use a `ConcurrentHashMap` here?

Reply:
Hmm no, I don't think that will be necessary. The same schema may be loaded twice but it would be pretty much harmless. On the other hand, if you change the `HashMap` to `ConcurrentHashMap`, we will have to pay the synchronization cost on each access to the map which is much much worst than the double schema loading.

Reply:
Ok, I'll revert it back :D

Reply:
Sorry, I didn't see you changed it already 😄